<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editBook</class>
 <widget class="QWidget" name="editBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Book</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icons/library.png</normaloff>:/icons/icons/library.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(244, 241, 222);
	color: rgb(61, 64, 91);
}
QPushButton
{
	font: 10pt &quot;IRANYekan&quot;;
	background-color: rgb(129, 178, 154);
	color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 255, 255);
    border-radius: 6px;
}
QPushButton:hover
{
	background-color: rgb(242, 204, 143);
}
QLineEdit
{
	font: 8pt &quot;IRANYekan&quot;;
}
QLabel
{
	font: 8pt &quot;IRANYekan&quot;;
}</string>
  </property>
  <widget class="QFrame" name="frame_top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(61, 64, 91);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/icons/edit-book.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>55</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 8pt &quot;IRANYekan Light&quot;;</string>
    </property>
    <property name="text">
     <string>Edit Form (Book) </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>25</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;IRANYekan&quot;;</string>
    </property>
    <property name="text">
     <string>Library Management System</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_isbn">
      <property name="text">
       <string>ISBN:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_isbn">
      <property name="inputMask">
       <string>000-0-00-000000-0</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_title_2">
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_title"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_author">
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_author"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_publisher">
      <property name="text">
       <string>Publisher:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_publisher"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_subject">
      <property name="text">
       <string>Subject:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_language">
      <property name="text">
       <string>Language:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_language"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_numpages">
      <property name="text">
       <string>Number Of Pages:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spinBox_numpages">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1600</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_available">
      <property name="text">
       <string>Number Of Availables:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBox_available"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_save">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;Save</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_subject"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_isbn</tabstop>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>lineEdit_author</tabstop>
  <tabstop>lineEdit_publisher</tabstop>
  <tabstop>lineEdit_subject</tabstop>
  <tabstop>lineEdit_language</tabstop>
  <tabstop>spinBox_numpages</tabstop>
  <tabstop>spinBox_available</tabstop>
  <tabstop>pushButton_save</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
